docs/source/sources.txt

```diff
@@ -143,6 +143,7 @@ rustworkx.is_isomorphic.html
 rustworkx.is_isomorphic_node_match.html
 rustworkx.is_matching.html
 rustworkx.is_maximal_matching.html
+rustworkx.is_strongly_connected.html
```

Comment:
Hmmm, I don't think this file does anything. You can notice by the fact we haven't updated it in nearly two years.

It must be from our old doc setup. I will investigate if we can remove it.

Reply:
Indeed, when we migrate the URL of our docs Matthew added a redirect step: https://github.com/Qiskit/rustworkx/blob/5f10121b0e4e9a8ee03b52bd69393164403fc0b9/docs/source/conf.py#L125

It doesn't hurt to be there, but it is mostly for old methods.

Reply:
If the file is not needed, then just remove it from the repo?

Reply:
Let's do it in a separate PR. It is no longer needed but there might be one or two places reading it. If we just remove it, we'll need to update the script